homeassistant/components/media_player/apple_tv.py

```diff
@@ -151,6 +159,22 @@ def _has_playing_media_changed(self, new_playing):
         return new_playing.media_type != old_playing.media_type or \
             new_playing.title != old_playing.title
 
+    def playstatus_error(self, updater, exception):
```

Comment:
Should be a ``@callback``, yes. It is called as a callback from an async method (think "future"), so everything it performs must be synced with event loop.